gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/FsBenchmark.java

```diff
@@ -201,6 +202,10 @@ private void benchmarkWrite(
     Set<LongSummaryStatistics> writeCallBytesList = newSetFromMap(new ConcurrentHashMap<>());
     Set<LongSummaryStatistics> writeCallTimeNsList = newSetFromMap(new ConcurrentHashMap<>());
 
+    // for all threads
+    ArrayList<Long> writeFileTime = new ArrayList<>();
+    ArrayList<Long> writeCallTime = new ArrayList<>();
```

Comment:
since items are added concurrently, having some form of concurrent list might be better. I understand that currently implementation has the issue.